docs/resources/policy_segment_port_profile_bindings.md

```diff
@@ -0,0 +1,105 @@
+---
+subcategory: "Segments"
+page_title: "NSXT: nsxt_policy_segment_port_profile_bindings"
+description: A resource to configure profile bindings for an existing Segment Port.
+---
+
+# nsxt_policy_segment_port_profile_bindings
+
+This resource provides a method for managing profile bindings (discovery, QoS, and security profiles) for an existing Segment Port. This resource is useful when you need to modify the profiles of a segment port that was created outside of Terraform or by another resource.
+
+~> **NOTE:** This resource modifies an existing segment port's profile bindings. It does not create a new segment port. The segment port must already exist.
```

Comment:
Please add another note, to avoid modifying ports which are created with `nsxt_policy_segment_port` with this resource.